eng/pipelines/mgmt.yml

```diff
@@ -6,7 +6,10 @@ pr:
     - '*-preview'
   paths:
     include:
+    - mgmt.proj
```

Comment:
Do we want to include the top level Directory.Build.props / targets in here as well?

Reply:
Yeah I guess we could include those as well I guess we just have to figure out how far we want to go with the list.

Reply:
I added in my follow-up PR.